pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -236,14 +225,108 @@ func initFSS(ctx context.Context, k8sClient clientset.Interface, controllerClust
 		}
 	}
 
-	// Set up kubernetes resource listeners for k8s orchestrator
+	var doesSvFssCRExist bool
+	if controllerClusterFlavor == cnstypes.CnsClusterFlavorGuest {
+		var isFSSCREnabled bool
+		// Check if csi-sv-feature-states-replication FSS exists and is enabled
+		if val, ok := k8sOrchestratorInstance.internalFSS.featureStates[common.CSISVFeatureStateReplication]; ok {
+			isFSSCREnabled, err = strconv.ParseBool(val)
+			if err != nil {
+				log.Errorf("Unable to convert %q to bool. csi-sv-feature-states-replication FSS disabled. Error: %v", val, err)
```

Comment:
start error msg with lower case.
`Unable to convert %q to bool` => `Unable to convert %v to bool`

Reply:
Fixed